site/content/troubleshooting-guide/index.md

```diff
@@ -67,4 +67,34 @@ AWS.Deploy.Tools, internally uses a variety of different services to host your .
 
 **Why is this happening**: This happens due to a know issue with the AWS Cloud Development Kit (CDK). The CDK is used to AWS.Deploy.Tools under the covers and it cannot cannot access the `$TEMP` directory inside the `$USERPROFILE` path if it contains a whitespace character.
 
-**Resolution**: See [here](../../docs/getting-started/custom-workspace) for guidance on setting a custom workspace that will be used by AWS.Deploy.tools.
\ No newline at end of file
+**Resolution**: See [here](../../docs/getting-started/custom-workspace) for guidance on setting a custom workspace that will be used by AWS.Deploy.tools.
+
+## AWS CDK Bootstrap related Deployment Failure
+
+A common error that is displayed for CDK Bootstrap related deployment failures is the following:
+```
+The AWS CDK Bootstrap, which is the process of provisioning initial resources for the deployment environment, has failed. Please review the output above for additional details [and check out our troubleshooting guide for the most common failure reasons]. You can learn more about CDK bootstrapping at https://docs.aws.amazon.com/cdk/v2/guide/bootstrapping.html.
+```
+The .NET Deployment Tool uses [AWS CDK](https://docs.aws.amazon.com/cdk/v2/guide/home.html) to create the AWS infrastructure needed to deploy your application. The AWS CDK is a framework for defining cloud infrastructure in code and provisioning it through AWS CloudFormation. 
```

Comment:
Replace ".NET Deployment Tool" with "AWS Deploy Tool for .NET"

Reply:
Done